modules/node-shared/src/test/scala/io/constellationnetwork/node/shared/infrastructure/consensus/MockValidationErrorStorage.scala

```diff
@@ -0,0 +1,19 @@
+package io.constellationnetwork.node.shared.infrastructure.consensus
+
+import java.time.Instant
+
+import cats.data.NonEmptyList
+import cats.effect.IO
+
+object MockValidationErrorStorage {
+
+  def make[Artifact, ValidationError](): ValidationErrorStorage[IO, Artifact, ValidationError] =
+    new ValidationErrorStorage[IO, Artifact, ValidationError] {
+      override def addValidationError(artifact: Artifact, errors: ValidationError): IO[Unit] = IO.unit
+
+      override def addValidationErrors(errors: NonEmptyList[(Artifact, ValidationError)]): IO[Unit] = IO.unit
+
+      override def allValidationErrors: IO[Vector[(Artifact, ValidationError, Instant)]] =
+        IO.pure(Vector.empty)
+    }
+}
```

Comment:
instead of a mock, you could use a normal storage for tests 🤔

Reply:
Done